internal/printer/customresourcedefinition.go

```diff
@@ -41,7 +41,7 @@ func CustomResourceDefinitionListHandler(
 		"We couldn't find any custom resource definitions!",
 		cols,
 		opts.DashConfig.ObjectStore())
-
+	ot.EnablePluginStatus(opts.DashConfig.PluginManager())
```

Comment:
Missing tests for this file.

Reply:
This is now explicitly tested under object_table_test.go